docs/Toy/Toy_Ch2.ipynb

```diff
@@ -6,29 +6,82 @@
    "id": "56341c66",
```

Comment:
<p>MLIR Syntax in Detail -&gt; Can we rephrase that so that it sounds like:</p><p>We give more details in the MLIR syntax? But not specify "detailed"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xdslproject/xdsl/pull/870/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Changed MLIR to IR, not sure exactly what you mean by not specifying "detailed". Do you mean that it's not really a detailed explanation?</p><p> </p>